pygame.py

```diff
@@ -0,0 +1,67 @@
+# Needed to use any and all python resources.
+import pygame
+import pygame_menu  #pip install pygame-menu
+from os import path
+from win32api import GetSystemMetrics 
+import yahtzee        
```

Comment:
I can't find the file where you created the yahtzee module. Or is that something that already exists in python? In this case I can't seem to find where you apply it (but maybe I am blind or it is in a file where there is no pull request on?).

Reply:
import yahtzee should import the other file called yahtzee.py. In this file, the whole game runs in the console. If you have both in the same working directory and run the classes.py first. Yathzee should just start to play.

Reply:
Hmm,  very good question... Have to look into it!:)